server_admin/topics/authentication/flows.adoc

```diff
@@ -342,29 +345,67 @@ claims= {
 
-The *option in the condition to store the achieved LoA in the user session* determines wheather the user will be asked to present the authentication factor again during a subsequent authentication. If true, the user will not be re-authenticated with this level during subsequent authentication. If false, it means that subsequent authentications will re-authentication with that level again. This is useful for operations that require higher security in the application (e.g. send payment) and always require authentication with the specific level.
+The option *Max Age* in the condition determines how long (how many seconds) the subsequent authentication level is valid. This helps to decide
+whether the user will be asked to present the authentication factor again during a subsequent authentication. If the particular level X is requested
+by the `claims` or `acr_values` parameter and the user has been already authenticated with level X, but it is expired (for example max age is configured to 300 and user authenticated before 310 seconds)
+then the user will be asked to re-authenticate again with the particular level. However, if the level is not yet expired, user will be automatically
+considered as authenticated with that level.
 
-.Implications about possible acr inputs in authentication request
-[cols="2",options="header"]
-|===
-| Authentication Request | Implication 
-| *No acr claim* is requested | Minimum authentication  (*1st Condition Flow*) is executed
-| *No essential claim* is requested in the acr claim | Default is *false*
-| *Unknown acr* value is requested | If essential false, minimum authentication is executed. If essential true, the outcome is a failed authentication attempt
-| User is re-authenticated via a *cookie* | Acr value in access token und ID token is `0`
-|===
+Using *Max Age* with value 0 means, that particular level is valid just for this single authentication. Hence every re-authentication requesting that level
+will need to authenticate again with that level. This is useful for operations that require higher security in the application (e.g. send payment) and always require authentication
+with the specific level.
+
+WARNING: Note that parameters like `claims` or `acr_values` might be changed by the user in the URL when the login request is sent from the client to the {project_name} via the user's browser.
+This can be mitigated if the client uses PAR (Pushed authorization request), a request object, or other mechanisms, which prevents the user from rewriting the parameters in the URL.
+Hence after the authentication, clients are encouraged to check the ID Token to doublecheck that `acr` in the token corresponds to the expected level.
+
+In case that there is no explicit level requested by parameters, the {project_name} will require the authentication with the first LoA
+condition found in the authentication flow (Username/Password in the example above). When a user was already authenticated with that level
```

Comment:
Change: flow (Username/Password in the example above)
TO
flow, such as in the preceding Username/Password example